changelog.md

```diff
@@ -15,6 +20,7 @@
 #### Deprecations
 
 * Deprecated property `CorrelatedStack.src`.
+* Reordered the keyword arguments of the method `CorrelatedStack.plot()` and enforced all parameters after `axes` to be keyword arguments. For details see the docstring.
```

Comment:
would be good to link to the actual page
```suggestion
* Reordered the keyword arguments of the method `CorrelatedStack.plot()` and enforced all parameters after `axes` to be keyword arguments. For details see the [docstring](https://lumicks-pylake.readthedocs.io/en/stable/_api/lumicks.pylake.correlated_stack.CorrelatedStack.html#lumicks.pylake.correlated_stack.CorrelatedStack.plot).
```

Reply:
👍